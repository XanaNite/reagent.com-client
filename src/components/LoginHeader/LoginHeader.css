.loginHeader h1{
    padding-left: 10px;
}

.loginHeader nav {
    padding-left: 10px;
    padding-bottom: 2px;
    background-color: rgb(37, 23, 71);
    color: seashell;
    text-align: right;
}

.loginHeader span{
    padding-left: 20px;
}

.loginHeader a{
    color: seashell;
    text-decoration: none;
    margin-right: 5px;
}