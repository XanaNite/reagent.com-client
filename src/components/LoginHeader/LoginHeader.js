import React from 'react';
import {Link} from 'react-router-dom';
import './LoginHeader.css';

export default function LoginHeader(){
    return(
        <header role="banner" className='loginHeader'>
            <nav role="navigation">
                <Link to='/register'>Sign Up</Link>
            </nav>
            <div className="signIn-title">
                <h1><Link to='/'>REagent.com</Link></h1>
                <span>for Professionals</span>
            </div>
        </header>
    );
}