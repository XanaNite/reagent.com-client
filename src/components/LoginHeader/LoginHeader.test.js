import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginHeader from './LoginHeader';

describe(`LoginHeader component`, () => {
    const props = {
        className: 'test-class-name',
        children: <p>test children</p>,
        'data-other': 'test-other-prop',
        agent: {
            first_name: "test",
            last_name: "test",
            title: "test",
            city: "test",
            state: "test"
        }
    };

    it('renders the LoginHeader given props', () => {
        const wrapper = shallow(<LoginHeader {...props} />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });
})