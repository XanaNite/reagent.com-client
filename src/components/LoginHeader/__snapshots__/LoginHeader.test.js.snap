// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginHeader component renders the LoginHeader given props 1`] = `
<header
  className="loginHeader"
  role="banner"
>
  <nav
    role="navigation"
  >
    <Link
      to="/register"
    >
      Sign Up
    </Link>
  </nav>
  <div
    className="signIn-title"
  >
    <h1>
      <Link
        to="/"
      >
        REagent.com
      </Link>
    </h1>
    <span>
      for Professionals
    </span>
  </div>
</header>
`;
